packages/cashier/src/pages/p2p-cashier/p2p-cashier.jsx

```diff
@@ -29,6 +29,7 @@ const P2PCashier = ({
     removeNotificationMessage,
     residence,
     setP2POrderProps,
+    setIsP2pPageShowing,
```

Comment:
Maybe `setIsP2pVisible` ?

Reply:
theres already a setIsP2pVisible, and thats for showing if the user can access p2p or nopt

Reply:
As discussed, we can change this to `setIsUserOnP2p`, thanks @ameerul-deriv 🙏